consensus/metrics.go

```diff
@@ -340,15 +370,20 @@ func NopMetrics() *Metrics {
 		MissingProposal: discard.NewGauge(),
 		RoundFailures:   discard.NewHistogram(),
 
-		ProposalCreating:        discard.NewHistogram(),
-		ProposalWaiting:         discard.NewHistogram(),
-		ProposalVerifying:       discard.NewHistogram(),
-		ProposalBlockReceiving:  discard.NewHistogram(),
-		PrevoteBlockVerifying:   discard.NewHistogram(),
-		PrevoteReceiving:        discard.NewHistogram(),
-		PrecommitBlockVerifying: discard.NewHistogram(),
-		PrecommitReceiving:      discard.NewHistogram(),
-		CommitBlockVerifying:    discard.NewHistogram(),
-		CommitBlockApplying:     discard.NewHistogram(),
+		DurationProposal:           discard.NewHistogram(),
+		DurationPrevote:            discard.NewHistogram(),
+		DurationPrecommit:          discard.NewHistogram(),
+		DurationCommitExecuting:    discard.NewHistogram(),
+		DurationCommitCommitting:   discard.NewHistogram(),
+		DurationCommitRechecking:   discard.NewHistogram(),
+		DurationWaitingForNewRound: discard.NewHistogram(),
+
+		DurationGaugeProposal:           discard.NewGauge(),
```

Comment:
About variable name: Why is `Gauge` added only to variable names whose type is Gauge? For example, it doesn't seem to be attached to Histgram type variables.

Reply:
There are `DurationProposal` and `DurationGaugeProposal`. `Grafana` needs a histogram, and Gauge is comfortable when investigating in person, so I put both.